src/ert/job_queue/driver.py

```diff
@@ -192,9 +263,52 @@ async def poll_statuses(self) -> None:
                 realstate.runend()
             if tokens[2] == "DONE" and str(realstate.current_state.id) == "RUNNING":
                 realstate.runend()
+            if tokens[2] not in LSF_STATUSES:
+                raise RuntimeError(
+                    f"The lsf_status {tokens[2]} for job {tokens[0]} was not recognized\n"
+                )
 
         self._currently_polling = False
 
-    async def kill(self, realization: "RealizationState") -> None:
-        print(f"would like to kill {realization}")
-        pass
+    async def kill(self, realization: "RealizationState"):
+        lsf_job_id = self._realstate_to_lsfid[realization]
+        logger.debug(f"Attempting to kill {lsf_job_id=}")
+        kill_cmd = [
+            self.get_option("BKILL_CMD") if self.has_option("BKILL_CMD") else "bkill",
+            lsf_job_id,
+        ]
+        assert shutil.which(kill_cmd[0])  # does not propagate back..
```

Comment:
I think this is due to me not understanding that asyncio errors are not necessarily sent to the console, but to logs/asyncio...txt. The line can be removed.